todo_app/README.md

```diff
@@ -0,0 +1,65 @@
+# DevOps Apprenticeship: To Do Checklist exercise-1
```

Comment:
Did you mean to update the README.md in the root of the project? You've mostly duplicated that and then added your extra section at the top.